Products/CMFPlone/tests/test_okay.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from plone.testing.z2 import Browser
+from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_INTEGRATION_TESTING
+
+import unittest
+
+
+class OkayTest(unittest.TestCase):
+    """Test the OK simple status view.
```

Comment:
Okay, I don't have a problem with that. But that's a blank line below the docstring, and you said above, which is what I didn't like.